tests/FsPickler.Tests/SerializerTests.fs

```diff
@@ -1197,6 +1201,34 @@ type SerializationTests (fixture : ISerializerFixture) =
                 let y = fixture.Serializer.Deserialize<int64[]> (f)
                 (y.Length, y.GetValue 0, y.GetValue (y.Length - 1))
 
+            let x = write()
+            GC.Collect(2, GCCollectionMode.Forced, true, true)
+            let y = read()
+            y |> should equal x
+        finally
+            System.IO.File.Delete path
+
+    // Large array tests output so much data they won't fit into a .NET byte array, we have to serialize to a FileStream.
+    [<Test; Explicit("Test takes many minutes to run"); Category("Stress tests")>]
```

Comment:
I made this explicit because it took about 10 minutes to run on my machine which felt a bit much for a "run every time test"